tmk_core/avr.mk

```diff
@@ -28,7 +28,7 @@ CFLAGS += -fno-inline-small-functions
 CFLAGS += -fno-strict-aliasing
 
 CXXFLAGS += $(COMPILEFLAGS)
-CXXFLAGS += -fno-exceptions -std=c++11
+CXXFLAGS += -fno-exceptions -std=c++14
```

Comment:
Is this change necessary? 

And if it is, does it compile on avr-gcc 5.4?

Reply:
Full-blown `constexpr` starts with C++14, so yes.

It does compile on 5.4, that's what I use.